modules/hal_nordic/nrfx/nrfx_config_nrf54h20_enga_ppr.h

```diff
@@ -0,0 +1,1877 @@
+#endif
+
+/**
+ * @brief NRFX_UARTE120_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE120_ENABLED
+#define NRFX_UARTE120_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE130_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE130_ENABLED
+#define NRFX_UARTE130_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE131_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE131_ENABLED
+#define NRFX_UARTE131_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE132_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE132_ENABLED
+#define NRFX_UARTE132_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE133_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE133_ENABLED
+#define NRFX_UARTE133_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE134_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE134_ENABLED
+#define NRFX_UARTE134_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE135_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE135_ENABLED
+#define NRFX_UARTE135_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE136_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE136_ENABLED
+#define NRFX_UARTE136_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_UARTE137_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_UARTE137_ENABLED
+#define NRFX_UARTE137_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_VEVIF_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_VEVIF_ENABLED
+#define NRFX_VEVIF_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_WDT_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_WDT_ENABLED
+#define NRFX_WDT_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_WDT_DEFAULT_CONFIG_IRQ_PRIORITY
+ *
+ * Integer value. Minimum: 0. Maximum: 3.
+ */
+#ifndef NRFX_WDT_DEFAULT_CONFIG_IRQ_PRIORITY
+#define NRFX_WDT_DEFAULT_CONFIG_IRQ_PRIORITY NRFX_DEFAULT_IRQ_PRIORITY
+#endif
+
+/**
+ * @brief NRFX_WDT_CONFIG_NO_IRQ - Remove WDT IRQ handling from WDT driver
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_WDT_CONFIG_NO_IRQ
+#define NRFX_WDT_CONFIG_NO_IRQ 0
+#endif
+
+/**
+ * @brief NRFX_WDT_CONFIG_LOG_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_WDT_CONFIG_LOG_ENABLED
+#define NRFX_WDT_CONFIG_LOG_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_WDT_CONFIG_LOG_LEVEL
+ *
+ * Integer value.
+ * Supported values:
+ * - Off     = 0
+ * - Error   = 1
+ * - Warning = 2
+ * - Info    = 3
+ * - Debug   = 4
+ */
+#ifndef NRFX_WDT_CONFIG_LOG_LEVEL
+#define NRFX_WDT_CONFIG_LOG_LEVEL 3
+#endif
+
+/**
+ * @brief NRFX_WDT131_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_WDT131_ENABLED
+#define NRFX_WDT131_ENABLED 0
+#endif
+
+/**
+ * @brief NRFX_WDT132_ENABLED
+ *
+ * Boolean. Accepted values: 0 and 1.
+ */
+#ifndef NRFX_WDT132_ENABLED
+#define NRFX_WDT132_ENABLED 0
+#endif
+
+#endif // NRFX_CONFIG_NRF54H20_ENGA_PPR_H__
```

Comment:
Compliance check will fail on this.
```suggestion
#endif /* NRFX_CONFIG_NRF54H20_ENGA_PPR_H__ */
```

Reply:
fixed all